content/download/data.en.md

```diff
@@ -1,39 +1,89 @@
----
-title: "Data"
-date: 2018-12-29T11:02:05+06:00
-description: "Download sample data ready for GRASS GIS."
-weight: 
-categories: [nl]
-layout: "data"
----
-
-  
-#### Quick links
-
-[ [**North Carolina dataset**](#NorthCarolinaDataset) | [**World dataset**](#WorldDataset) | [**Spearfish dataset**](#SpearFishDataset) ]
-
-<div class="alert rounded-0 alert-default">
-<i class="fa fa-arrow-right"></i> This section gathers sample datasets available as ready-to-use GRASS GIS <b>locations</b> and common GIS formats.
-</div>
-
-<hr>
-
-### <a name="NorthCarolinaDataset"></a> North Carolina data set
-
-
-<hr>
-
-
-### <a name="WorldDataset"></a> World dataset
-
-
-<hr>
-
-
-### <a name="SpearFishDataset"></a> Spearfish dataset
-
-
-<hr>
-
-
-
+---
+title: "Data"
+date: 2018-12-29T11:02:05+06:00
+description: "Download sample data ready for GRASS GIS."
+weight: 
+categories: [nl]
+layout: "data"
+---
+
+  
+#### Quick links
+
+[ [**North Carolina dataset**](#NorthCarolinaDataset) | [**Spearfish dataset**](#SpearFishDataset) | [**Datasets for special topics**](#SpecialTopicDatasets) ]
+
+<div class="alert rounded-0 alert-default">
+<i class="fa fa-arrow-right"></i> This section gathers sample datasets available as ready-to-use GRASS GIS <b>locations</b> and other common GIS formats.
+</div>
+
+<div class="alert rounded-0 alert-default">
+<i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the download button on the GRASS GIS startup screen.
+</div>
+
+<hr>
+
+### <a name="NorthCarolinaDataset"></a> North Carolina data set
+
+This dataset is a modern package of geospatial data from North Carolina, USA. It offers raster, vector, LiDAR and satellite data.
+Most of the examples in the module manuals are written with this dataset. Furthermore, the tessuite operates with this dataset.
+The description, list of maps, and a quick usage tutorial are available here.
+The North Carolina dataset can be downloaded in two versions:
+<ul id="links" class="list-unstyled version">
+ <li>
+  <span class="mwl"><a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip " target="_blank"> nc_spm_full_v2alpha </a></span>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip" class="inl btn btn-primary" target="_blank">Download ZIP (159MB)</a>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz" class="inl btn btn-secondary">Download TAR.GZ (159MB)</a>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" target="_blank"> nc_basic_spm_grass7 </a></span>

```

Comment:
I would rename it the Simple (or Basic) dataset

Reply:
Yes, to keep it consistent "North Carolina basic dataset"